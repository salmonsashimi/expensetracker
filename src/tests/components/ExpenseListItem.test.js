import React from 'react';
import { shallow } from 'enzyme';
import ExpenseListItem from '../../components/ExpenseListItem';
import expenseArray from '../fixtures/expenseArray'


test('should render ExpenseListItem correctly', () => {
    const wrapper = shallow(<ExpenseListItem {...expenseArray[0]} />);
    expect(wrapper).toMatchSnapshot();

})