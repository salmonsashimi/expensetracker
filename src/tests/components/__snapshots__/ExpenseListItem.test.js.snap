// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseListItem correctly 1`] = `
<div>
  <Link
    to="/edit/1"
  >
    <h3>
      Water Bill
       
    </h3>
  </Link>
  <p>
    $500.00
    ,
    1st January 1970
  </p>
</div>
`;
